docs/developer/events.md

```diff
@@ -0,0 +1,70 @@
+*Copied from https://gist.github.com/mootari/d39895574c8deacc57d0fc04eb0d21ca*
+*Original author: https://github.com/mootari*
+
+# Events
```

Comment:
What about adding this to the `Event` TSDoc comment. You could also add links to the other events listed.